Fixed p2p section

effofxprime opened the pull request:
I went through, fixed up a bit of the grammar/wording of a few things.
Updated the styling so that code will line wrap, that way all code can be seen before copying it.
For now, I have only updated MAINNET, TODO: TESTNET.

effofxprime committed:
Fixed p2p section

Fixed the grammar a bit, added the seed section and tried to clean up things to be more crisp and short.

effofxprime committed:
state-sync instructions

Added barebone instructions for getting a statesync setup.

Corey-Code commented on docs/getting-started/mainnet.md:
```diff
@@ -206,22 +218,22 @@ You will see JSON output where you need to locate **_catching_up_** field. When
 :::
 
-In order to create validator you need to have Vidulum account and some funds, whcih can be delegated to validator.
+In order to create a validator, you must have a Vidulum wallet and at least 1 VDL that you must self delegate as the validator.
```
Change to at least 2 VDL to cover TX fee when creating a validator. Best practice would be at least 10 VDL but this would ultimately be up to the user

effofxprime replied:
[9e0e007](https://github.com/vidulum/documentation/pull/3/commits/9e0e0074a2c84f5b7794e96f15f9ab64c408f8d4)
Fixed.

Corey-Code reviewed (approved):
Looks great

effofxprime committed:
Updated for state sync, pruning, sentrys

This update corrects the RPC server listed.  I found mixing the seed and rpc server caused occasional latency that resulted in a missed block.

Trust period and block's to keep was re-evaluated.  2/3 time of 21 days is ~14 days.  Thus, my time and block calculations are based off that.

Maintenance section was updated and I added State Sync, Pruning, and Sentry setup instructions

effofxprime committed:
Removed sentry peer # suggestion

The peer # suggestion has been re-thought and, keeping the defaults here is better.
I am adding more than 3, and was conflating some of my own personal goals/thoughts towards the docs without realizing it.  And even then, I've gone back for myself and am using default 40/10 peers.

effofxprime committed:
Removed Maintenance additions for commits

I removed all the additions to separate the commits.  That way, if you just want to first add the stuff about mainnet and state sync for a new validator, you can.  Just trying to give options.

effofxprime committed:
Maintenance additions

These are all the maintenance additionns.
These includes:
Pruning
State Sync
Sentry Install

effofxprime commented:
Morning!  Okay, I have updated what was changed on my end (IE: the rpc server) in the instructions for state sync.  Commit 0326b00 will just contain the edits for all currently viewable documentation.  It just contains the initial validator setup instructions with state sync.

bd7aba3, this part will contain all of the maintenance additions that I have added.
- State Sync for current validators
- Pruning for current validators
- Running Sentry's

I have a live feed of this documentation that you can read here:  http://erialosofastora.asuscomm.com:8080/docs/getting-started/

effofxprime commented:
I just saw the comment above to change to 2VDL.  Let me fix that, Sorry about that.

effofxprime committed:
mainnet fix - validator required start qty

Changed the recommended quantity to 2vdl as requested.

Corey-Code commented on docs/getting-started/maintenance.md:
````diff
@@ -146,4 +146,368 @@ In output look for following line:
+
+As your `vidulum` user:
+```bash
+vidulum unsafe-reset-all
````
Should be vidulumd unsafe-reset-all

effofxprime replied:
Live working copy to view: http://erialosofastora.asuscomm.com:8080/

effofxprime committed:
Fixed typo - Removed State sync in validator creation

I fixed the typo for the 'vidulumd' command and also removed state sync from the initial validator setup.

effofxprime committed:
Added additional warning

The 'unsafe-reset-all' also reset's a nodes ability to detect if it is in risk of double signing or not.
I felt this should be mentioned so no one accidentally screws themselves over.

effofxprime commented:
[4892f35](https://github.com/vidulum/documentation/pull/3/commits/4892f353791e1cda874abfebbcfc9b12633572c2)

I felt it best to add the warning regarding `unsafe-reset-all` when it comes to how the priv_state.json file holds the validators last signed block info.

effofxprime committed:
RPC snapshot correction

After filing an issue on github about snapshots failing to create, I was made aware that I was creating them too soon.
A snapshot is now made ever 3000 blocks.
It was recommended to do it only every 250k.
https://github.com/cosmos/cosmos-sdk/issues/9686#issuecomment-1064271936

With this correction, the snapshot's are taken every 3000 blocks, and 100 are kept.  So 300k blocks can be restored in the event of some unforeseen circumstance.
